python/tvm/runtime/profiler_vm.py

```diff
@@ -50,14 +50,17 @@ def get_stat(self, sort_by_time=True):  # pylint: disable=unused-argument
         warnings.warn("get_stat has been removed, use profile instead")
         return ""
 
-    def profile(self, *args, func_name="main", **kwargs):
+    def profile(self, *args, func_name="main", collectors=[], **kwargs):
```

Comment:
Interesting pylint didn't complain about this one.

Reply:
can you fix this one too?